akka-remote/src/main/scala/akka/remote/Remoting.scala

```diff
@@ -146,6 +145,7 @@ private[remote] class Remoting(_system: ExtendedActorSystem, _provider: RemoteAc
           endpointManager = None
         }
 
+        import system.dispatcher
```

Comment:
Yes, I think the caller of this shutdown() method actually blocks on an Await, so this is probably safe.